### DISCUSSION TITLE
站点不稳定完全无法工作，请求退款。

### DISCUSSION THREAD

**Discussion by 1342509884zc** [getlantern/lantern-forum-cn#995] [Q&A]
一分钟就自动切几个站点，我的脸书账号都被风控了，完全无法工作，请求退款!!!!!
我需要能固定站点梯子!!!3天了邮件无人答复，为了工作已经买了别家，请求退款谢谢！！
![e1cbd8ea93448a6b2006bda117676ec](https://github.com/user-attachments/assets/f3f71db2-94fa-4fb0-b812-3c37cd9512fd)

**Comment by MaxMKMax** [comment]
@1342509884zc 您好. 请您提供我帐号邮箱. 我帮您提交上去. 谢谢.
